eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -69,8 +69,12 @@ case class NodeParams(nodeKeyManager: NodeKeyManager,
                       toRemoteDelay: CltvExpiryDelta,
                       maxToLocalDelay: CltvExpiryDelta,
                       minDepthBlocks: Int,
-                      feeBase: MilliSatoshi,
-                      feeProportionalMillionth: Int,
+                      feeBasePublic: MilliSatoshi,
+                      feeProportionalMillionthPublic: Int,
+                      feeBasePrivate: MilliSatoshi,
+                      feeProportionalMillionthPrivate: Int,
+                      minTrampolineFeeBase: MilliSatoshi,
+                      minTrampolineFeeProportionalMillionth: Int,
```

Comment:
Let's define some classes in the `Relayer` companion object:

```scala
case class RelayFees(feeBase: MilliSatoshi, feeProportionalMillionth: Int)
case class RelayParams(publicChannelFees: RelayFees,
                       privateChannelFees: RelayFees,
                       minTrampolineFees: RelayFees)
``` 

```suggestion
                      relayParams: RelayParams,
```